pkg/manager/impl/dataset_manager_test.go

```diff
@@ -39,6 +40,23 @@ func getTestDataset() *datacatalog.Dataset {
 	}
 }
 
+func getDatasetModel(dataset datacatalog.Dataset) models.Dataset {
```

Comment:
why not use the transformer instead of adding this method?

Reply:
I'll do that. In tests I usually like to avoid using the actual code, but I guess since it's tested we can trust it here.

Reply:
oh got it, makes sense